### PR TITLE
Update V2 ISO wording

### PR DESCRIPTION
**Pull request by Aubron**
Updates the language of endpoint deprecation, for v2ISO

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by jmulford-bandwidth** [comment]
Can you also update the file here?

https://github.com/Bandwidth/v2-messaging/blob/master/methods/createMessage.md

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://v2-messaging-DX-219.s3-website-us-east-1.amazonaws.com/

**Comment by Aubron** [comment]
Good catch @jmulford-bandwidth

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://v2-messaging-DX-219.s3-website-us-east-1.amazonaws.com/

File: methods/createMessage.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Endpoint for sending text messages and picture messages using V2 messaging.
 
 <code class="post">POST</code>`https://messaging.bandwidth.com/api/v2/users/{userId}/messages`
 
-** Notice: The old URL `https://api.catapult.inetwork.com/v2/users/{userId}/messages` will be deprecated June 13th, 2019. Please update your system to use the new URL. **
+** Notice: The old URL `https://api.catapult.inetwork.com/v2/users/{userId}/messages` has been deprecated, and will be turned off June 13th, 2019. Please update your system to use the new URL. **
```
Comment (tato123) [inline methods/createMessage.md]: Should turned off say `taken offline` given that the other two changes use that language?

Comment (Aubron) [reply methods/createMessage.md]: I _can_, but you all have to re-review it. Captive audience

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://v2-messaging-DX-219.s3-website-us-east-1.amazonaws.com/

File: methods/createMessage.md (hunk 1 above)
Comment (tato123) [reply methods/createMessage.md]: I'm fine with turned off, my question was mainly whether that was specific expected language

**Pull request merged**